### PR TITLE
WIP Docs updates

### PR DESCRIPTION
**Pull request by jryanhaber**

### CODE REVIEW

File: doc/holochain_101/src/faq.md [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
+6.  [Can you run a cryptocurrency on Holochain?](#can-you-run-a-cryptocurrency-on-holochain)
+7.  [How is Holochain different from \***\*\_\_\*\***?](#how-is-holochain-different-from-__________)
+8.  [What language is Holochain written in? What languages can I use to make Holochain apps?](#what-language-is-holochain-written-in-what-languages-can-i-use-to-make-holochain-apps)
+9.  [Is Holochain open source?](#is-holochain-open-source)
+    10 [How is Holochain more environmentally ethical than blockchain?](#how-is-holochain-more-environmentally-ethical-than-blockchain)
+10. [How are data validated on Holochain?](#how-are-data-validated-on-holochain)
+11. [What happens to data when a node leaves the network?](#what-happens-to-data-when-a-node-leaves-the-network)
+12. [Should I build my coin/token on Holochain?](#should-i-build-my-cointoken-on-holochain)
+13. [What does “agent-centric” mean? How is this different from “data-centric”?](#what-does-agent-centric-mean-how-is-this-different-from-data-centric)
+14. [What is the TPS (Transactions Per Second) on Holochain?](#what-is-the-tps-transactions-per-second-on-holochain)
```
Comment (sphinxc0re) [inline doc/holochain_101/src/faq.md]: Could you change all these to `1.`
The markdown renderer is taking care of enumeration itself

File: doc/holochain_101/src/faq.md [hunk 2]
```diff
@@ -124,15 +121,15 @@ However, if your system is designed around data having one absolute true state,
...
 
-2. There's no mining on Holochain. Blockchain's proof-of-work system provides a hefty incentive for thousands of people to spend the processing power of their CPUs and GPUs using up [huge amounts](https://digiconomist.net/bitcoin-energy-consumption) [of electricity](https://motherboard.vice.com/en_us/article/ywbbpm/bitcoin-mining-electricity-consumption-ethereum-energy-climate-change) on solving a meaningless cryptographic puzzle. Holochain doesn't have mining.
+2.  There's no mining on Holochain. Blockchain's proof-of-work system provides a hefty incentive for thousands of people to spend the processing power of their CPUs and GPUs using up [huge amounts](https://digiconomist.net/bitcoin-energy-consumption) [of electricity](https://motherboard.vice.com/en_us/article/ywbbpm/bitcoin-mining-electricity-consumption-ethereum-energy-climate-change) on solving a meaningless cryptographic puzzle. Holochain doesn't have mining.
 
-## How is Holochain different from __________?
+## How is Holochain different from \***\*\_\_\*\***?
```
Comment (sphinxc0re) [inline doc/holochain_101/src/faq.md]: I don't understand this change. Please elaborate.

File: doc/holochain_101/src/faq.md (hunk 1 above)
Comment (jryanhaber) [reply doc/holochain_101/src/faq.md]: Thank you for commenting.  This work in progress is muddy due to the unanticipated effects of my javascript linter.  I plan to recommit these changes while keeping an eye on that.

File: doc/holochain_101/src/faq.md (hunk 2 above)
Comment (jryanhaber) [reply doc/holochain_101/src/faq.md]: This was an attempt to take this one small step forward, but it has been clarified that the format prior was intentional, so I will not recommit this suggestion when I circle back to these commits.

**Comment by jryanhaber** [comment]
Im closing this pull request since I resubmitted the changes in separate pull requests while making sure my linter did not create unanticipated side effects.

**Pull request closed without merging**